libraries/Microsoft.Bot.Builder.Core.Extensions/FileStorage.cs

```diff
@@ -38,9 +38,9 @@ public Task Delete(string[] keys)
             return Task.CompletedTask;
         }
 
-        public async Task<IEnumerable<KeyValuePair<string, object>>> Read(string[] keys)
+        public async Task<IDictionary<string, object>> Read(string[] keys)
         {
-            var storeItems = new List<KeyValuePair<string, object>>(keys.Length);
+            IDictionary<string, object> storeItems = new Dictionary<string, object>(keys.Length);
```

Comment:
The codebase seems to be `var` based, so I think we would want to stay consistent with that.

Reply:
Fixed. Thanks. I must have been reading too much Java code recently!!